jenkins/job-configs/global.yaml

```diff
@@ -7,6 +7,10 @@
                     mkdir -p _tmp
                     curl -fsS --retry 3 "https://raw.githubusercontent.com/kubernetes/kubernetes/master/hack/jenkins/upload-to-gcs.sh" > ./_tmp/upload-to-gcs.sh
```

Comment:
Yeah, that's certainly awkward. We retry this 3 times and we retry the upload several times, but if github is down or something we'll still get no log where we would otherwise.

I don't know if there's a good solution, really.